hydromt_wflow/wflow.py

```diff
@@ -28,6 +28,8 @@
 from pyflwdir import core_conversion, core_d8, core_ldd
 from shapely.geometry import box
 
+from hydromt_wflow.utils import mask_raster_from_layer
+
```

Comment:
If you look 1 and 3 lines below we import .utils or things from utils a lot here. Is it best to use relative import or hydromt_wflow ie .utils or hydromt_wflow.utils?
I'm not sure what is the best here but would be happy if you could harmonise with what you think is best :)